modules/brainbrowser/js/brainbrowser.volume-viewer.min.js

```diff
@@ -1,31 +1,887 @@
 /*
-* BrainBrowser: Web-based Neurological Visualization Tools
-* (https://brainbrowser.cbrain.mcgill.ca)
-*
-* Copyright (C) 2011
-* The Royal Institution for the Advancement of Learning
-* McGill University
-*
-* This program is free software: you can redistribute it and/or modify
-* it under the terms of the GNU Affero General Public License as
-* published by the Free Software Foundation, either version 3 of the
-* License, or (at your option) any later version.
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-* GNU Affero General Public License for more details.
-*
-* You should have received a copy of the GNU Affero General Public License
-* along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+ * BrainBrowser: Web-based Neurological Visualization Tools
```

Comment:
Revert these changes, this is a minified version and should be in that format. This code base is external to ours and should not be modified here.